gdb-xtensa-esp32-elf/meta.yaml

```diff
@@ -3,7 +3,7 @@
 
 package:
   name: {{ name }}
-  version: {{ version }}.memfault0
+  version: {{ version }}.memfault1
```

Comment:
Is this needed in the case that someone has the current version of this package installed? Otherwise Conda won't know to update the package?

Reply:
yeah, we can't (or shouldn't?) just overwrite the existing one, i think, even though they're broken for the s3

Reply:
unlike pypi, anaconda.org doesn't seem to have a notion of "pulled", beyond just deleting the package. maybe not that important, though